ydb/core/scheme/scheme_tablecell.h

```diff
@@ -10,79 +10,148 @@
 #include <util/system/unaligned_mem.h>
 #include <util/memory/pool.h>
 
+#include <bit>
 #include <deque>
 #include <type_traits>
 
 namespace NKikimr {
 
-#pragma pack(push,4)
+static_assert(std::endian::native == std::endian::little, "TCell expects little endian architecture for data packing");
+
 // Represents one element in a tuple
 // Doesn't own the memory buffer that stores the actual value
-// Small values (<= 8 bytes) are stored inline
+// Small values (<= 14 bytes) are stored inline
 struct TCell {
```

Comment:
Do we need some alignas?
Before your changes it was 8?
But maybe it's ok if we never read with assumption that this data is aligned